.gitignore

```diff
@@ -1,3 +1,6 @@
+.vagrant
+harvester_tracker.json
```

Comment:
There are some import failures in HarvesterConfig

Reply:
And pta-intelligent-search-harvester/src/test/java/fi/maanmittauslaitos/pta/search/metadata/BestMatchingRegionListCustomExtractorTest.java

Reply:
These should be fixed